features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/domain/ResourceRequestBodyFactory.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.domain
+
+import androidx.annotation.VisibleForTesting
+import com.datadog.android.api.InternalLogger
+import com.datadog.android.sessionreplay.internal.recorder.SessionReplayResource
+import com.datadog.android.sessionreplay.internal.recorder.SessionReplayResourceContext
+import com.google.gson.JsonIOException
+import okhttp3.MediaType.Companion.toMediaTypeOrNull
+import okhttp3.MultipartBody
+import okhttp3.RequestBody
+import okhttp3.RequestBody.Companion.toRequestBody
+
+internal class ResourceRequestBodyFactory {
+    fun create(
```

Comment:
It seems that this method is useless, the only thing it does it the call to another method with exactly the same arguments. We need either to remove this one or inline the body.